hw/dv/data/sim.mk

```diff
@@ -72,6 +72,8 @@ ifneq (${sw_name},)
 		${run_dir}/rom.32.vmem
 	cp ${sw_build_dir}/build-out/sw/device/${sw_dir}/${sw_name}_${sw_build_device}.32.vmem \
 		${run_dir}/sw.32.vmem
+	cp ${sw_build_dir}/build-out/sw/device/${sw_dir}/${sw_name}_${sw_build_device}.64.vmem \
+		${run_dir}/sw.64.vmem
```

Comment:
@sriyerg 
@weicaiyang 

just copying over all of them.

Reply:
Ack, fine with me.

Reply:
is the `sw.32.vmem` still needed?

Reply:
hmmm good point i can remove it. the 32 version is only needed for ROM right now.

Reply:
@sriyerg 
when you add support to override the 32/64 from dvsim, will you also create a make variable so we can reference it below? So we don't have to hard code 32/64.  I'm guessing that's what you have planned?

Reply:
yeah i think that would be a good idea. I don't really think in the makefiles we should have hardcoded information about word sizes etc.